### PR TITLE
Add simplified Chinese and traditional Chinese language

### PR DESCRIPTION
**Pull request by MaLaGeBe**

### CODE REVIEW

**Comment by MaLaGeBe** [comment]
bro, it has been modified as you said

File: locales/tc.lua [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  -- Weapon Tints
+  ['tint_default'] = '默認色調',
+  ['tint_green'] = '綠色調',
+  ['tint_gold'] = '金色調',
+  ['tint_pink'] = '粉色調',
+  ['tint_army'] = '軍用色調',
+  ['tint_lspd'] = '洛聖都警察局色調',
+  ['tint_orange'] = '橘色調',
+  ['tint_platinum'] = '柏金色調',
+}
```
Comment (ElPumpo) [inline locales/tc.lua]: missing trailing whitespace under this line (222)

Comment (MaLaGeBe) [reply locales/tc.lua]: OK, it has been modified!

**Comment by ElPumpo** [comment]
Translate new strings too then ill merge

**Comment by ElPumpo** [comment]
Also add the new locales for the new command

**Comment by ElPumpo** [comment]
Ok so since last time I've added additional translations, please do the final two strings for each locale and we're ready to merge.

**Comment by MaLaGeBe** [comment]
ok

**Comment by ElPumpo** [comment]
Ok man thanks for your help!

**Pull request merged**